tests/xdebug-filter.php

```diff
@@ -33,13 +33,13 @@
 
 declare(strict_types=1);
 
-if (!function_exists('xdebug_set_filter')) {
+if (!\function_exists('xdebug_set_filter')) {
     return;
```

Comment:
same here

Reply:
It seems like that's the cost of having this ruleset work in other places. It just won't make them imported here and there.